.zshrc

```diff
@@ -0,0 +1,6 @@
+export LANG=en_US.UTF-8
```

Comment:
@simiyu1 This file is not necessary on GitHub, please remove it. Also, it exposes private data.

Reply:
On it!

Reply:
Resolved